mujoco_ros_msgs/srv/GetEqualityConstraintParameters.srv

```diff
@@ -0,0 +1,6 @@
+string name
+string admin_hash
+---
+mujoco_ros_msgs/EqualityConstraintParameters parameters
```

Comment:
Is name here duplicate? Because `EqualityConstraintParameters` already contains a name field.